packages/openapi-react-query/src/index.ts

```diff
@@ -141,10 +142,11 @@ export default function createClient<Paths extends {}, Media extends MediaType =
             const mth = method.toUpperCase() as Uppercase<typeof method>;
             const fn = client[mth] as ClientMethod<Paths, typeof method, Media>;
             const { data, error } = await fn(path, init as InitWithUnknowns<typeof init>);
-            if (error || !data) {
+            if (error) {
               throw error;
             }
-            return data;
+
+            return data as Exclude<typeof data, undefined>;
```

Comment:
shouldn't matter since the type is dictated by return type of `createClientz anyway, its not inferred from here - just checked that what we return is valid